alvr/common/Cargo.toml

```diff
@@ -36,23 +30,26 @@ nonzero_ext = "0.2"
 tokio = { version = "1", features = ["rt", "net", "macros"] }
 tokio-util = { version = "0.6", features = ["codec", "net"] }
 # Graphics
-gfx-hal = "=0.6.0"
-winit = "0.24" # needed to get the screen size
+egui = "0.13"
+wgpu = "0.9"
+winit = "0.25"     # needed to get the screen size
 # Audio
 cpal = "0.13"
-rodio = "0.13"
+rodio = "0.14"
 # Miscellaneous
-rand = "0.8"
+alvr_filesystem-layout = { path = "../filesystem-layout" }
 encoding_rs_io = "0.1"
+nalgebra = { version = "0.28", features = ["serde-serialize"] }
+rand = "0.8"
 rcgen = "0.8"
+semver = { version = "1", features = ['serde'] }
```

Comment:
Reminder: Need to test if the serialized data format changes because of the bump.

Reply:
True

Reply:
Tested [here](https://github.com/ckiee/alvr-743-check), it is fine both in debug and release modes.